chrome-developer-tools/chrome-developer-tools.md

```diff
@@ -27,22 +27,24 @@ Chrome Developer Tools help tremendously with front-end development (HTML, CSS &
 - How to access the JavaScript console to run and debug code
 - Participants will learn the various keyboard shortcuts to open Chrome Developer Tools
 
-### Supplemental Materials
+### Materials
+- [Chrome Developer Tools (video walkthrough of slides)](https://drive.google.com/file/d/1LV6lOI0MRKUVSaDlSipiy6JX8EtEaYjH/view?usp=sharing)
+- [Chrome Developer Tools (slides)](https://docs.google.com/presentation/d/1z8aIzOxV5L-zW3MZmZhX67DgyLq_4ErO4yTS4CIp7R0/edit?usp=sharing)
```

Comment:
@vsharma20v Please make sure the presentation follows all the formatting guidelines mentioned in #600 . As I am not having the edit access of the slideshow so I can't review it.